src/core/utils/server.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+import path from 'path';
+
+import fs from 'fs-extra';
+import config from 'config';
+import type { $Request, $Response } from 'express';
+
+import log from 'core/logger';
+
+const EXCLUDES_LIST = [
```

Comment:
> I wonder if we should move feature flags to a namespace (object config parameter) first to avoid the white-list

Hmm, yes, that would make it a lot easier to expose the flag parameters but would it make things awkward for overriding? How would you do it? Current you can do this:

https://github.com/mozilla/addons-frontend/blob/40d1ba4e8fe83bfdcf4bc44ba3854a5ba8a4ea61/config/dev-amo.js#L29

It's nice that you don't have to worry about other flags in that one line.

However, maybe it would also be nice to *explicitly* see the entire object of feature flags getting set for each env? I'm not sure.

Reply:
I think I can make a change in `getFakeConfig()` to allow:

```js
getFakeConfig({ 'feature_flags.enableInlineAddonReview': true });
```

but that might introduce a bit of complexity..

An alternative would be to find a different prefix for feature flags (other than `enable`).

Reply:
`enableFeature` as a prefix looks like it would still keep the variables readable.